src/components/Help/Steppers/UserMountsStepper.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import {
+    Stepper,
+    Step,
+    StepLabel,
+    Typography,
+    Button,
+} from '@material-ui/core';
+import { USER_MOUNTS_STEPPER_STEPS, USER_MOUNTS_STEPPER_INSTRUCTIONS } from 'utils/constants';
+import 'components/Help/Steppers/Stepper.scss';
+import UserMountsInstructionsSelectUser from 'utils/Assets/UserMountsInstructionsSelectUser.gif';
+import UserMountsInstructionsChangeOwned from 'utils/Assets/UserMountsInstructionsChangeOwned.gif';
+import UserMountsInstructionsConfirmChanges from 'utils/Assets/UserMountsInstructionsConfirmChanges.gif';
+import UserMountsInstructionsExpansionFilter from 'utils/Assets/UserMountsInstructionsExpansionFilter.gif';
+
+function getStepContent(stepIndex) {
```

Comment:
Please use ES6 arrow function syntax.
```
const getStepContent = (stepIndex) => {...}
```

Reply:
all functions not use ES6 arrow function syntax